src/attaching.js

```diff
@@ -77,6 +80,18 @@ export function isOutOfContext(origin, context) {
   return !contextView;
 }
 
+export function isStaleContent(origin, transferStartedAt) {
+  transferStartedAt = parseInt(transferStartedAt);
```

Comment:
This code is in the same module, so this is a minor point, but it seems weird that `isStaleContent` accepts the param `transferStartedAt` as a string and then has to parse it itself. It would make a little more sense for this function to accept either an integer (number of milliseconds since reference date), a Date, or the view that has the `data-transfer-started-at` attribute. The reason why the value needs to be parsed is b/c it was pulled from an HTML data attribute, but the function is disconnected from that fact b/c it happened elsewhere.